Automattic/wp-calypso
# Editing Toolkit: Fix IE11 syntax error, new dependencies to transpile

#### Changes proposed in this Pull Request
These dependencies aren't being transpiled which causes `syntax error` in IE 11. This PR adds both dependencies to the list of modules that should be transpiled.

* `dom7` (https://github.com/nolimits4web/dom7/issues/21)
* `swiper` (https://github.com/nolimits4web/swiper/issues/3285)

#### Testing instructions
Testing locally
* run `yarn dev`
* cd to `wp-calypso/apps/editing-toolkit`
* run `yarn wp-env start`
* You should be able to access a WordPress instance at `localhost:4013`
* Verify block editor works in IE11 and no errors in the console
